apps/gdalbuildvrt_lib.cpp

```diff
@@ -208,7 +208,8 @@ static int  GetSrcDstWin(DatasetProperty* psDP,
         *pdfSrcYSize = *pdfDstYSize / dfSrcToDstYSize;
     }
 
-    return TRUE;
+    return *pdfSrcXSize >= 0 && *pdfDstXSize >= 0 &&
+           *pdfSrcYSize >= 0 && *pdfDstYSize >= 0;
```

Comment:
I'm surprised this is >= 0 instead of > 0 since it was the size=0 sources that was triggering an error later.

Reply:
> I'm surprised this is >= 0 instead of > 0 since it was the size=0 sources that was triggering an error later.

ah indeed, should be > 0...  will fix. This part is more a safety belt. The actual change that fixes the issue is the changes at top of the function